.circleci/config.yml

```diff
@@ -31,13 +31,15 @@ jobs:
           name: Install New Updated Package
           command: |
             brew install --build-from-source Casks/circleci-runner.rb 2> err.out || echo "possible cask method error on install"
+            cat err.out
```

Comment:
This was difficult to debug initially because the `brew install ...` command swallowed the error. This ensures we can inspect the error in the step output.

Reply:
I'm more in favor of storing it as an artifact, but I do not have strong feelings about it

Reply:
I figure folks would be less apt to check the artifact. I didn't even realize stderr was being redirected at first.